src/txn/txn_rollback_to_stable.c

```diff
@@ -202,9 +201,8 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
     newer_hs_durable_ts = unpack->tw.durable_start_ts;
 
     /* Open a history store table cursor. */
-    WT_ERR(__wt_hs_cursor_open(session));
-    hs_cursor = session->hs_cursor;
-    cbt = (WT_CURSOR_BTREE *)hs_cursor;
+    WT_ERR(__wt_curhs_open(session, NULL, &hs_cursor));
+    F_SET(hs_cursor, WT_CURSTD_HS_READ_COMMITTED);
```

Comment:
My impression is no need for us to set READ_COMMITTED explicitly, when we perform RTS, we make sure no active txns are in the system. so ideally we don't need to set this. if any test is failing I think we need to understand why? if we need to set, then I think it needs a comment about the special case why we need to set it as READ_COMMITTED.

Reply:
Fair enough. Let me run some tests with flag removed and see how it goes.

Reply:
I have added comments to explain why we need to set this flag for rollback-to-stable use.